tests/lib/pages/login_page.rb

```diff
@@ -6,10 +6,11 @@ class LoginPage<BasePage
   button(:submit_login_form, name:'login')
   link(:lost_password, css:'#login-form td a')
 
-  def login (user)
+  def login
```

Comment:
am I right that we're not able to call login with an argument if we need?
I would suggest supporting both solutions
but I'm not sure
you can leave it like this I think